packages/six.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Setuptools < Package
```

Comment:
Dude, please don't add stuff without testing it first.  You are WAY too fast.  You need to slow down and test before spewing out random commits.  This class does not match the six.rb filename.

Reply:
Sorry. Fixed that. I'm sort of stressed to get this fixed. I have something to do tomorrow. It's done.

Reply:
Maybe we should just shelve this until you are ready?  It can wait. :)

Reply:
I think I'm good now.